guava/src/com/google/common/math/IntMath.java

```diff
@@ -99,8 +105,9 @@ public static boolean isPowerOfTwo(int x) {
    * a signed int. The implementation is branch-free, and benchmarks suggest it is measurably (if
    * narrowly) faster than the straightforward ternary expression.
    */
+  @SuppressWarnings("return.type.incompatible")// lessThanBranchFree() is specified to return 1 if x < y and 0 otherwise
```

Comment:
Can you add prefix `value:`?

Reply:
The comment on a suppressed warning should explain why the code is correct. No need to repeat the specification which is right before.